frontend/src/containers/LoginView.tsx

```diff
@@ -14,87 +14,88 @@ interface DispatchToProps {
 }
 
 interface StateToProps {
-  loading: boolean,
-  error: Error,
+    loading: boolean,
+    error: Error,
 }
 
 type Props = RouteComponentProps & DispatchToProps & StateToProps & FormComponentProps;
 
-class NormalLoginForm extends React.Component<Props> {
-  handleSubmit = (e: React.FormEvent) => {
-    e.preventDefault();
-    this.props.form.validateFields((err, values) => {
-      if (!err) {
-        this.props.authLogin(values.username, values.password);
-      }
-    });
-    this.props.history.push('/');
-  };
-
-  render() {
+const LoginForm = (props: Props) => {
```

Comment:
this component's name is `LoginForm`, and it sits in a file called `LoginView`, that is nestend in folder that says `containers`. You see where I'm going with this? :D